web/eslint.config.js

```diff
@@ -51,7 +51,15 @@ export default tseslint.config(
             ],
             "@typescript-eslint/no-unused-vars": [
                 "error",
-                { argsIgnorePattern: "^_" },
+                {
+                    args: "all",
+                    argsIgnorePattern: "^_",
+                    caughtErrors: "all",
+                    caughtErrorsIgnorePattern: "^_",
+                    destructuredArrayIgnorePattern: "^_",
+                    varsIgnorePattern: "^_",
+                    ignoreRestSiblings: true,
```

Comment:
Is this something we actually want, or just something that would be too tedious to fix now?

Reply:
Bah, I was specifically referring to `ignoreRestSiblings`. Why didn't it pick only that line... The rest (haha, puntended) I can understand, sure.

Reply:
Oh, good question! I think so yes.

The syntax is this:
```typescript
let {x, y, ...rest} = {x: 1, y: 2, z: 3, w: 4};
console.log(x); // 1
console.log(y); // 2
console.log(rest); // Object { z: 3, w: 4 }
```

If you want to extract part of an object using the rest syntax, that's how you (can) do it. You'd need to ignore the the ones you specify, and then you can keep the ones you don't. Without this set to true, you need to do something like `let {x: _x, y: _y, ...rest} = ....`

I don't think we actually use this today though.

Reply:
Oh, I see. It's probably fine to set it then.